### PR TITLE
Fix typo in populate command: option first-page

### PR DESCRIPTION
**Pull request by bujnos**

### CODE REVIEW

File: src/Command/PopulateCommand.php [hunk 1]
```diff
@@ -139,7 +139,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $options['last_page'] = $input->getOption('last-page');
         }
 
-        if ($input->isInteractive() && $reset && 1 < $input->getOption('first_page')) {
+        if ($input->isInteractive() && $reset && 1 < $input->getOption('first-page')) {
```
Comment (clement-michelet) [inline src/Command/PopulateCommand.php]: You stored the value of `$input->getOption('first-page')` in `$options['first_page']`. 
Why not re-use that value ?

Comment (bujnos) [reply src/Command/PopulateCommand.php]: A good point, I changed it.

**Comment by XWB** [comment]
Thanks.

**Comment by trsteel88** [comment]
I think we need a new 4.1.1 tag for this?

**Pull request merged**